src/pages/case-studies/codecov-uses-sourcegraph-to-resolve-incidents-faster.tsx

```diff
@@ -0,0 +1,266 @@
+import { FunctionComponent } from 'react'
+
+import {
+    Layout,
+    NewCaseStudyJumbotron,
+    NewCaseStudyLayout,
+    ContentSection,
+    Blockquote,
+    ThreeUpText,
+    UseChallengeSolutionResults,
+} from '@components'
+
+const threeUpTextItems = [
+    {
+        subtitle: <h3 className="pb-3 mx-md-auto display-3 font-weight-bold text-blurple">5 minutes</h3>,
+        description:
+            "Codecov reviewed their entire codebase and confirmed that they weren't exposed to the Log4j vulnerability.",
+    },
+    {
+        subtitle: <h3 className="pb-3 mx-md-auto display-3 font-weight-bold text-blurple">12x faster</h3>,
+        description:
+            'Codecov was able to resolve Log4j 12x faster than with their code host\'s native search functionality and report absolute confidence in that resolution to their customers.',
+    },
+    {
+        subtitle: <h3 className="pb-3 mx-md-auto display-3 font-weight-bold text-blurple">100% confidence</h3>,
+        description:
+            'The security team at Codecov is able to complete code reviews independently and with complete confidence that their work is correct.',
+    },
+]
+
+export const CaseStudy: FunctionComponent = () => (
+    <Layout
+        meta={{
+            title: 'Codecov uses Sourcegraph to resolve incidents faster',
+            description:
+                'Codecov case study. Learn how Codecov uses Sourcegraph to resolve incidents like Log4j 12x faster and with 100% confidence.',
+        }}
+        heroAndHeaderClassName="bg-code-venus"
```

Comment:
If you merge main in, you can remove this and use the Background component. This currently loads an 8 MB SVG 😬

Reply:
Ok I opened up `ProductHero` here [2be13bb](https://github.com/sourcegraph/about/pull/5592/commits/2be13bb67436bd74ea15ba037e342c04baef6183) to accommodate these heros. The Product heros still look good to me with this change

Reply:
That's wonderful! Thanks for doing that @st0nebraker! Do you want to remove that 8 MB svg? 🔥

Reply:
I should've mentioned, it's still being used in `GatedResourceLayout.tsx` right now. So may be best to address the rest of it in #5633 ?

Reply:
No worries! Yeah we can address the rest in #5633!